src/patterns/ec2-app.ts

```diff
@@ -34,6 +64,24 @@ export enum GuApplicationPorts {
   Play = 9000,
 }
 
+const OpenApplicationCidrError: Error = Error(`Your list of CIDR ranges includes 0.0.0.0/0, meaning all other ranges specified are unnecessary as
+your application is open to the world. Please either remove the open CIDR range or the other specified ones.`);
+
+function checkValidCidrRanges(access: RestrictedAccess) {
+  const openToWorld = access.cidrRanges.map((range) => range.uniqueId).includes("0.0.0.0/0");
+  const moreThanOneRange = access.cidrRanges.length > 1;
+
+  if (openToWorld && moreThanOneRange) throw OpenApplicationCidrError;
```

Comment:
I don't think we need to check if there is `moreThanOneRange` here. Specifying `RestrictedAccess` with `"0.0.0.0/0"` simply doesn't make sense so we should always throw an error in that scenario in my opinion.

Reply:
Correct, that's absolutely right! Good catch